matminer/utils/data.py

```diff
@@ -224,6 +224,45 @@ def get_elemental_property(self, elem, property_name):
     def get_oxidation_states(self, elem):
         return self.all_elemental_props["OxidationStates"][elem.symbol]
 
+class MeredigData(AbstractData):
```

Comment:
This module seems redundant with the MagpieData and PymatgenData. Unless I am mistaken, could you eliminate this class and use one of the existing databases instead?

Reply:
My reasoning for adding a new module was that there was already significant overlap between MagpieData, PymatgenData, and DemlData, and so I had assumed that one featurizer -> one data retrieval class was a style choice. I can change to use one of the existing classes if this isn't the case?

Reply:
Ah, that makes sense. We do not have a requirement of one data class per featurizer, but do have many almost-exactly-similar classes to err on the side exactly reproducing the featurizers from those papers. Since the Meredig features use the same data as the Magpie ones, you could be get away with using MagpieData for your featurizers.

For full context, I would be happiest if we could merge all of the data classes into one but have not had the time to figure out how to do that effectively.

Reply:
Changed to use Magpie data source